Tests/Tests/STPCardTest.m

```diff
@@ -191,66 +237,118 @@ - (void)testCardEquals {
     STPCard *card1 = [STPCard decodedObjectFromAPIResponse:[self completeAttributeDictionary]];
     STPCard *card2 = [STPCard decodedObjectFromAPIResponse:[self completeAttributeDictionary]];
 
-    XCTAssertEqualObjects(card1, card1, @"card should equal itself");
-    XCTAssertEqualObjects(card1, card2, @"cards with equal data should be equal");
+    XCTAssertNotEqual(card1, card2);
+
+    XCTAssertEqualObjects(card1, card1);
+    XCTAssertEqualObjects(card1, card2);
+
+    XCTAssertEqual(card1.hash, card1.hash);
+    XCTAssertEqual(card1.hash, card2.hash);
 }
 
 #pragma mark - Description Tests
 
-- (void)testDescriptionWorks {
+- (void)testDescription {
     STPCard *card = [STPCard decodedObjectFromAPIResponse:[self completeAttributeDictionary]];
     XCTAssert(card.description);
 }
 
 #pragma mark - STPAPIResponseDecodable Tests
 
 - (NSDictionary *)completeAttributeDictionary {
+    // Source: https://stripe.com/docs/api#card_object
     return @{
-             @"id": @"1",
-             @"exp_month": @"12",
-             @"exp_year": @"2013",
-             @"funding": @"debit",
-             @"name": @"Smerlock Smolmes",
-             @"address_line1": @"221A Baker Street",
-             @"address_city": @"New York",
-             @"address_state": @"NY",
-             @"address_zip": @"12345",
-             @"address_country": @"USA",
-             @"last4": @"1234",
-             @"dynamic_last4": @"5678",
-             @"brand": @"MasterCard",
-             @"country": @"Japan",
+             @"id": @"card_1AVRojEOD54MuFwSxr93QJSx",
```

Comment:
Where do these ids actually come from? I have different ones on my screen. Are we sure these aren't "real" tokens that are populated from the logged in Stripe account or something?

Also again I think all fake json stuff should be through STPFixtures now.